cmd/kubemark/hollow-node.go

```diff
@@ -92,6 +95,9 @@ func (c *hollowNodeConfig) addFlags(fs *pflag.FlagSet) {
 	bindableNodeLabels := cliflag.ConfigurationMap(c.NodeLabels)
 	fs.Var(&bindableNodeLabels, "node-labels", "Additional node labels")
 	fs.Var(utiltaints.NewTaintsVar(&c.RegisterWithTaints), "register-with-taints", "Register the node with the given list of taints (comma separated \"<key>=<value>:<effect>\"). No-op if register-node is false.")
+	fs.IntVar(&c.MaxPods, "max-pods", maxPods, "Number of pods that can run on this Kubelet.")
+	bindableExtendedResources := cliflag.ConfigurationMap(c.ExtendedResources)
+	fs.Var(&bindableExtendedResources, "extended-resources", "Register the node with extended resources (common separated \"<name>=<quantity>\")")
```

Comment:
common -> comma ?

Reply:
Nice catch. This is a typo and I will fix it